From: Josenilson Ferreira da Silva <nilsonfsilva@hotmail.com>
Date: Tue, 30 Apr 2024 06:14:47 +0800
Subject: Prefer fromtimestamp over utcfromtimestamp

Forwarded: not-needed
Last-Update: 2024-02-16

===================================================================
---
 whey/builder.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/whey/builder.py b/whey/builder.py
index d94caaf..c6a4ca2 100644
--- a/whey/builder.py
+++ b/whey/builder.py
@@ -59,6 +59,8 @@ from shippinglabel.requirements import ComparableRequirement, combine_requiremen
 # this package
 from whey import additional_files
 
+from datetime import timezone
+
 __all__ = ("AbstractBuilder", "SDistBuilder", "WheelBuilder")
 
 archive_name_sub_re = re.compile(
